beacon8/layers/SpatialSubSampling.py

```diff
@@ -0,0 +1,14 @@
+from .Module import Module
+
+
+class SpatialSubSampling(Module):
+    """
+    note that it behaves very differently from Torch!
+    """
+    def __init__(self, scale):
+        self.scale = scale
+
+    def symb_forward(self, symb_input):
+        if symb_input.ndim != 4:
+            raise NotImplementedError
+        return symb_input[:, :, ::self.scale, ::self.scale]
```

Comment:
As soon as Theano/Theano#2487 lands, we can make this generic.

If you want, using [this trick](https://github.com/lucasb-eyer/DeepFried/blob/master/layers.py#L810) `scale` could be either an integer it is now, or a tuple for different scales for the dimensions, and then use it as `symb_input[:, :, ::self.scale[0], ::self.scale[1]]`.